app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ Required ] I think it would be nice if the styling for this `login` form is similar to the styling provided on the project wireframe. Currently, your design varies from the wireframe in the following ways.
   
   - [ ] The `Login button` should have the same width as the `email` and `password` fields 
   - [ ] The `email` and `username` input fields should have `placeholder` texts instead of the labels.

## Your design
![Screenshot from 2023-10-18 23-36-45](https://github.com/melkamu12/microverse_blogapp/assets/16392046/d6782df0-28a1-4372-8027-f759f6999f94)

## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)

Reply:
thank you i consider it.